pkgs/development/python-modules/gpy/default.nix

```diff
@@ -19,6 +19,14 @@ buildPythonPackage rec {
 
   buildInputs = [ cython ];
 
+  # Force cython-generated .c files to be re-built since the included
+  # ones were built with an older version of cython that produces .c
+  # files which are incompatible with python3.9
+  preConfigure = ''
+    echo "[build_ext]" >> setup.cfg
+    echo "force=1" >> setup.cfg
+  '';
+
```

Comment:
It seems other modules are using the following? Is this the same functionality?
```
  preBuild = ''
    ${python.interpreter} setup.py build_ext
  '';
```

Reply:
Just tried it -- apparently no. I do think another tactic that would work is just removing the generated .c files, which would force cython to re-translate them.

Reply:
My two cents: removing pre-generated *.c files in `preBuild` is probably easier to understand, but I am fine with the proposed change too.

Reply:
Agreed. Simply removing them didn't do it though, but I force-pushed a new version that just regenerates the cython files ourselves in preBuild.